### ISSUE TITLE
Gogle Fonts Lato not loading lato-ext properly (on Chrome!!! [Win7 64bit chrome version 66])

### ISSUE THREAD

**Issue by OndraX** [google/fonts#1564]
I am having some difficulty with Google Fonts on Windows 7 getting extended Latin characters (ěščřžů in particular) to load from the Google Webfonts served version of Lato. I am loading the font within a custom Wordpress theme using the following line:

"      wp_enqueue_style( 'lato-font','https://fonts.googleapis.com/css?family=Lato:100,300,400,700&amp;subset=latin-ext',false,'1.1','all');
![image](https://user-images.githubusercontent.com/32552330/40063938-c0d835ae-585e-11e8-8fc2-72ef04d7c81e.png)

yet 

 (note the ž for example works fine.)

Doing some digging around in the Dev Console, I see there is a problem with the files which are being loaded.

![image](https://user-images.githubusercontent.com/32552330/40064030-f65ce576-585e-11e8-8cf1-adf9e5d19296.png)

clearly some of the _p.woff files are the latin-ext ones, and all of the _p.woff files just load nothing:

![image](https://user-images.githubusercontent.com/32552330/40064124-2956217c-585f-11e8-8fe4-9d710f638a82.png)

**Comment by OndraX** [comment]
Right, thanks for the quick reply.
They do, however, promise the `latin-ext` character set and the font does exist in a variant which supports these characters. The fact the there even is a tickmark for the character set (and all major Central and Eastern European languages' characters are supposed to be even in the smallest subset of `latin-ext`) seems a bit of a betrayal.

**Comment by padinko** [comment]
Hi,
I got this issue too. 

I tried to download Webfonts from http://www.latofonts.com/ and use it. This webfonts have all this characters. So maybe problem is only in google fonts?

**Comment by OndraX** [comment]
I agree, though I had to download the LatoLatin font rather than just Lato
with an extended character set. I just think this is an annoying quirk of
Google Fonts + I didn't really know whom to address concerning the issue,
as there seem to be help forums for all Google products but Webfonts.



2018-05-23 15:48 GMT+02:00 padinko <notifications@github.com>:

**Comment by vyskoczilova** [comment]
Hi, any update on this?

**Comment by MarcelGerber** [comment]
I am also experiencing this problem with the Lato and Roboto fonts.
One solution is to explicitly load Lato with `&text=≥≤`, but this is not a particularly great solution. I found no other way (subset or stuff like that) to load these glyphs.

A demo can be found here: https://codesandbox.io/s/google-fonts-lato-greater-than-equal-p5icv

**Comment by mfranzs** [comment]
Bumping this as well :)

**Comment by andmade** [comment]
I'm seeing this issue as well. The lato-ext font seems to be rendering properly in Firefox, but not in Chrome or Safari. It's causing a ğ to render in the fallback font (Arial) rather than in Lato, thus looking conspicuously out of place.

**Comment by forusak** [comment]
funny, it's 2022, 4 years later and it's still not fixed

**Comment by brendon** [comment]
Yep, missing āēīōū in all font variants regular and below.

**Comment by chantalboado** [comment]
Same problem.

Googling a lot I found the page of https://developers.google.com/fonts/docs/css2 and this solved my problem.

**Comment by stovmascript** [comment]
The new `css2` URL didn't solve anything for me.

I'm actually trying to migrate from Adobe Fonts (Typekit), where Lato works properly, to Google Fonts because we also use Material Icons.

I'm using the latest snippets from GF:

```
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
```

```
font-family: 'Lato', sans-serif;
```

**Comment by pulyabacsi** [comment]
Found a solution!
I had the same problem with Hungarian characters like áéüöúőű
the above mentioned new type of include links are needed BUT I missed the most basic thing, usually not mentioned anywhere I researched the problem

**Solution > you need to add the character coding to the head:**
_Ensure that your HTML document is using the correct character encoding. The most common and recommended encoding is UTF-8, which supports a wide range of characters, including Latin extended characters. 
You can specify the character encoding in the <head> section of your HTML document using a meta tag:_
`<meta charset="UTF-8">`

**Comment by kratos-digital** [comment]
It's not working for me. I have meta charset defined.

![Screenshot 2024-03-16 at 14 24 40](https://github.com/google/fonts/assets/144851446/360377c5-73c6-4996-be68-87df5b712b83)

You can see in the screenshot that the `c` letter is totally different font, some default sans-serif.

Using this code provided by Google Fonts:
`https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap`

There was Latin Extended checkbox before, now there isn't such option.

Terrible new UX with dropped features...
